melos.yaml

```diff
@@ -3,22 +3,17 @@ repository: https://github.com/flame-engine/flame
 
 packages:
   - packages/**
-  - examples/**
+  - examples/
+  - examples/games/**
   - tutorials/**
   - doc/**
 
 command:
-  bootstrap:
-    # Uses the pubspec_overrides.yaml instead of having Melos modifying the lock file.
-    usePubspecOverrides: true
-
   version:
-    # Generate commit links in package changelogs.
-    linkToCommits: true
     # Only allow versioning to happen on main branch.
     branch: main
-    # Additionally build a changelog at the root of the workspace.
-    workspaceChangelog: true
+    # Generates a link to a prefilled GitHug release creation page.
```

Comment:
I could use a GitHug right now :)

Reply:
Haha when your code is so good it feels like a hug 😄